common/vlog.h

```diff
@@ -13,9 +13,10 @@ namespace Carbon {
 //
 // For example:
 //   CARBON_VLOG() << "Verbose message";
-#define CARBON_VLOG()                 \
-  (vlog_stream_ == nullptr) ? (void)0 \
-                            : CARBON_VLOG_INTERNAL_STREAM(vlog_stream_)
+#define CARBON_VLOG()                             \
+  __builtin_expect(vlog_stream_ == nullptr, true) \
```

Comment:
Looks like there's an issue merging. This is at head, but it's showing as part of the edits in this PR. FWIW, looking at the commit history, I'd probably do something like `git rebase HEAD~ --onto trunk`, but there are probably also other ways to get a proper merge commit.